src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+    FormControl,
+    FormHelperText,
+    InputLabel,
+    MenuItem,
+} from "@mui/material";
+import React from "react";
+import { DropdownType } from "../../Constants/application.type";
+import { SelectInput } from "./Dropdown.styles";
+
+const formControlDefaultProps = {
+    fullWidth: true,
+};
+
+const selectDefaultProps = {
+    variant: "filled",
+    fullWidth: true,
+    textcolor: "black",
+    label: "inputText",
+};
+
+export const Dropdown: React.FC<any> = React.forwardRef(
+    (props: DropdownType, ref) => (
+        <FormControl
```

Comment:
Can we destructure the props?

Ex :  const { helperText, options, label } = props;

Reply:
Agreed! will change it.

Reply:
done